File: 1945-sum-of-digits-of-string-after-convert/1945-sum-of-digits-of-string-after-convert.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        int res=0;
        string ss=""; 
        
        for(int i=0;i<s.size();i++){
            int n= s[i]-'a'+1;
            ss+=  to_string(n);
        }
       while(k--){
           res=0;
        for(int i=0;i<ss.size();i++){
        res+=ss[i]-'0';
            
        }
       ss=to_string(res);
       }
        
        return res; 
        
        
        
        
    }
};

File: 1945-sum-of-digits-of-string-after-convert/1945-sum-of-digits-of-string-after-convert.js
/**
 * @param {string} s
 * @param {number} k
 * @return {number}
 */
var getLucky = function(s, k) {
    let res=0;
        let ss=""; 
        
        for(let i=0;i<s.length;i++){
           let n = s.charCodeAt(i) - 'a'.charCodeAt(0) + 1;
           let g = n.toString();
           ss += g;
        }
       while(k--){
           res=0;
        for(let i=0;i<ss.length;i++){
        res+=ss[i]-'0';
            
        }
        
       ss=res.toString();
       }
        
        return res; 
        
       
    
};